packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -545,11 +556,11 @@ export class SpinButton extends React.Component<ISpinButtonProps, ISpinButtonSta
     }
   };
 
-  private _onIncrementMouseDown = (): void => {
-    this._updateValue(true /* shouldSpin */, this._initialStepDelay, this._onIncrement!);
+  private _onIncrementMouseDown = (event: React.MouseEvent<HTMLElement>): void => {
```

Comment:
Should this be?
```suggestion
  private _onIncrementMouseDown = (event?: React.MouseEvent<HTMLElement>): void => {
```

Reply:
Since this is directly passed to the onMouseDown, it is required if you don't just omit it altogether.